src/index.html

```diff
@@ -4,9 +4,314 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <div class="header">
+        <img src="./images/logo.png" alt="Moyo logo">
+        <nav class="header__nav">
+          <ul class="header__ul">
+            <li class="header__li">
+              <a href="#" class="header__link is-active">Apple</a>
+            </li>
+            <li class="header__li">
+              <a class="header__link" href="#">Samsung</a>
+            </li>
+            <li class="header__li">
+              <a class="header__link" href="#">Smartphones</a>
+            </li>
+            <li class="header__li">
+              <a
+                class="header__link"
+                href="#"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="header__li">
+              <a class="header__link" href="#">Gadgets</a>
+            </li>
+            <li class="header__li">
+              <a class="header__link" href="#">Tablets</a>
+            </li>
+            <li class="header__li">
+              <a class="header__link" href="#">Photo</a>
+            </li>
+            <li class="header__li">
+              <a class="header__link" href="#">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <div data-qa="card" class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        >
+        <div class="card__name">
+          <h1 class="card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <div class="card__code">Product code: 195434</div>
+        </div>
+        <div class="card__feedback">
+          <div class="stars">
```

Comment:
```suggestion
          <div class="stars stars--4">
```
The number of active stars should depend on the modifier